nx-X11/programs/Xserver/fb/Imakefile

```diff
@@ -1,22 +1,3 @@
-/*
```

Comment:
Leave out "(, yet)" in your commit message, you already included a yet earlier. :)